english/new-years/new-years.md

```diff
@@ -1,2 +1,4 @@
 Ah . . . New Year's Eve . . . it all comes back to you now . . . sort of. 
-It was a costume party, and, come to think of it, the Chief Marshmallow was there too.
\ No newline at end of file
+It was a costume party, and, come to think of it, the Chief Marshmallow was there too.
+
+He was obviously angry about something, it could be the horrible smell that was coming from the [raccoon in the corner](raccoon-corner/raccoon-corner.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.